Mathlib/RingTheory/PowerSeries/Inverse.lean

```diff
@@ -149,7 +149,6 @@ theorem constantCoeff_inv (φ : k⟦X⟧) : constantCoeff k φ⁻¹ = (constantC
 theorem inv_eq_zero {φ : k⟦X⟧} : φ⁻¹ = 0 ↔ constantCoeff k φ = 0 :=
   MvPowerSeries.inv_eq_zero
 
-@[simp]
```

Comment:
Are you removing `simp` because of the linter?

Reply:
Yeah, the `MvPowerSeries` versions are `simp`'d and now the discrimination tree can see them.